src/main/scala/com/fulcrumgenomics/vcf/HapCutToVcf.scala

```diff
@@ -117,6 +123,32 @@ class HapCutToVcf
     vcfWriter.close()
   }
 
+  /** If fixing ambiguous (IUPAC) bases is desired, returns a new path to a VCF, otherwise returns the original input VCF. */
+  private def inputVcf: FilePath = if (!fixAmbiguousReferenceAlleles) this.vcf else {
+
+    val inputLines = PathUtil.extensionOf(this.vcf) match {
+      case Some(".vcf") => Io.readLines(this.vcf)
+      case Some(".vcf.gz")  =>
+        val stream = Io.toInputStream(this.vcf)
+        Source.fromInputStream(stream).withClose(() => stream.close()).getLines()
+      case Some(".bcf") => throw new IllegalArgumentException(s"BCF not supported when fixing ambiguity codes in: ${this.vcf}")
+      case _            => throw new IllegalArgumentException(s"Could not determine file type from ${this.vcf}")
+    }
+    val outputLines = inputLines.map { line =>
+      if (line.startsWith("#")) line else {
+        val items = line.split('\t')
+        val bases = items(3).map { base =>
```

Comment:
Two thoughts - does this handle both ref and alts?  I.e. what if you had a deletion or insertion containing ambiguity codes?  Secondly, do you need to make sure you're not in a symbolic allele like `<REF>`?

Reply:
Yup, wasn't handling alts, but now we are.  Symbolic alleles are skipped, I forgot about those.